src/components/Message/SingleMessage.tsx

```diff
@@ -0,0 +1,79 @@
+import styled, { css } from 'styled-components';
+import { IMessageData, IUserNameProps } from '../../interface/interface';
+const SingleMessage = ({ chat }: { chat: IMessageData }) => {
+  const { userId, userName, text, time } = chat;
+  return (
+    <SingleMessageContainer userName={userName}>
+      <ProfileImg src={`/assets/${userId}.jpg`} />
+      <ContextContainer>
+        <UserName userName={userName}>{userName}</UserName>
+        <MessageText userName={userName}>{text}</MessageText>
+      </ContextContainer>
+      <ShowTime>{time}</ShowTime>
+    </SingleMessageContainer>
+  );
+};
+
+export default SingleMessage;
+
+const SingleMessageContainer = styled.div<IUserNameProps>`
+  padding: 1rem;
+  display: flex;
+  ${({ userName }) =>
+    userName === 'chaaerim'
+      ? css`
+          flex-direction: row-reverse;
+        `
+      : css`
+          justify-content: flex-start;
+        `}
```

Comment:
이렇게 하면 더 간결하게 쓸 수 있습니다

```suggestion
    flex-direction: ${({ userName }) =>
    userName === 'chaaerim' ? 'row-reverse' : 'row'};
```

Reply:
ㄷ ㅐ박..8줄이 2줄이 되는 매직이네요..  더 공부해봐야겠어요 !!!